Packs/AzureSecurityCenter/Integrations/AzureSecurityCenter_v2/AzureSecurityCenter_v2.py

```diff
@@ -220,19 +70,377 @@ def normalize_context_key(string):
+            full_url = f"{self.server}/providers/Microsoft.Management/managementGroups/{management_group}"
+            full_url += cmd_url
+            return self.ms_client.http_request(method="GET", full_url=full_url, url_suffix="", params=params)
+        return self.ms_client.http_request(method="GET", url_suffix=cmd_url, params=params)
+
+    def list_jit(self, asc_location, resource_group_name):
+        """Building query
+
+        Args:
+            asc_location: Machine location
+            resource_group_name: Resource group name
+
+        Returns:
+            dict: response body
+        """
+        params = {"api-version": JIT_API_VERSION}
+        cmd_url = f"/resourceGroups/{resource_group_name}" if resource_group_name else ""
+        cmd_url += f"/providers/Microsoft.Security/locations/{asc_location}" if asc_location else ""
+        cmd_url += "/providers/Microsoft.Security/jitNetworkAccessPolicies"
+        return self.ms_client.http_request(method="GET", url_suffix=cmd_url, params=params)
+
+    def get_jit(self, policy_name, asc_location, resource_group_name):
+        """Building query
+
+        Args:
+            policy_name: Policy name
+            asc_location: Machine location
+            resource_group_name: Resource name group
+
+        Returns:
+            dict: response body
+        """
+        cmd_url = f"/resourceGroups/{resource_group_name}/providers/Microsoft.Security/locations/{asc_location}/" \
+                  f"jitNetworkAccessPolicies/{policy_name}"
+        params = {"api-version": JIT_API_VERSION}
+
+        return self.ms_client.http_request(method="GET", url_suffix=cmd_url, params=params)
+
+    def initiate_jit(self, resource_group_name, asc_location, policy_name, vm_id, port, source_address, duration):
+        """Starting new Just-in-time machine
+
+        Args:
+            resource_group_name: Resource group name
+            asc_location: Machine location
+            policy_name: Policy name
+            vm_id: Virtual Machine ID
+            port: ports to be used
+            source_address: Source address
+            duration: Time in
+
+        Returns:
+            dict: response body
+        """
+        cmd_url = f"/resourceGroups/{resource_group_name}/providers/Microsoft.Security/locations/{asc_location}/" \
+                  f"jitNetworkAccessPolicies/{policy_name}/initiate"
+        params = {"api-version": JIT_API_VERSION}
+
+        # only supports init access for one vm and one port now
+        data = {
+            "virtualMachines": [
+                {
+                    "ID": vm_id,
+                    "ports": [
+                        {
+                            "number": port,
+                            "duration": duration,
+                            "allowedSourceAddressPrefix": source_address,
+                        }
+                    ],
+                }
+            ]
+        }
+        # response code should be 202 Accepted
```

Comment:
shouldnt we pass it in `ok_code` then ?

Reply:
ill add it. but I think we should keep the comment because its the only request with this response code and it describes why I changed the resp_type (as with 204 code)